core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation.directive.ts

```diff
@@ -43,15 +43,24 @@ angular.module('oppia').directive('editorNavigation', [
       controller: [
         '$rootScope', '$scope', '$timeout', '$uibModal', 'ContextService',
         'ExplorationFeaturesService', 'ExplorationRightsService',
-        'ExplorationWarningsService', 'RouterService',
+        'ExplorationWarningsService', 'ImprovementTaskService', 'RouterService',
         'SiteAnalyticsService', 'StateTutorialFirstTimeService',
         'ThreadDataService', 'UserService', 'WindowDimensionsService',
         function(
             $rootScope, $scope, $timeout, $uibModal, ContextService,
             ExplorationFeaturesService, ExplorationRightsService,
-            ExplorationWarningsService, RouterService,
+            ExplorationWarningsService, ImprovementTaskService, RouterService,
             SiteAnalyticsService, StateTutorialFirstTimeService,
             ThreadDataService, UserService, WindowDimensionsService) {
+          var taskCount;
+          ImprovementTaskService.fetchOpenTaskCount().then(taskCountResp => {
```

Comment:
Do not abbreviate (I don't know what "resp" is). Call it countOfOpenTasks perhaps.

Reply:
I have made the necessary changes.